### ISSUE TITLE
bviplus: Does not build

### ISSUE THREAD

**Issue by matthiasbeyer** [NixOS/nixpkgs#15270]
## Issue description

`bviplus` does not build.

### Steps to reproduce

`nix-shell -p bviplus`

## Technical details

* System: 16.09.git.9510670 (Flounder)
* Nix version: nix-env (Nix) 1.11.2
* Nixpkgs version: "16.09pre82105.7465bcd"

**Comment by matthiasbeyer** [comment]
@zimbatm @cstrahan

**Comment by zimbatm** [comment]
What's the output ?

**Comment by zimbatm** [comment]
```
vf_backend.h:40:13: warning: inline function 'compute_percent_complete' declared but never defined
```

**Comment by joachifm** [comment]
Could be due to new inline semantics. Similar issues have been fixed by either `-fgnu89-inline` or specifying a pre C11 std.

**Comment by zimbatm** [comment]
Fixed by #16464
